src/scripting/lua_game.cc

```diff
@@ -667,21 +667,26 @@ int LuaPlayer::reveal_fields(lua_State* L) {
 }
 
 /* RST
-   .. method:: hide_fields(fields[, unexplore = false])
+   .. method:: hide_fields(fields[, state = "seen"])
 
       Undo the effect of :meth:`reveal_fields` on these fields for the current player and
-      optionally completely hide them.
+      optionally completely hide them or reset them to unexplored.
       See also :ref:`field_animations` for animated hiding.
+      *Note*: hide_fields(fields[, unexplore = false]) is deprecated and will be converted to
+              ``"permanent"`` if true and ``"seen"`` if false.
```

Comment:
```suggestion
      .. note:: Passing a :class:`boolean` as the *state* argument is deprecated.
         Use ``"permanent"`` instead of ``true`` and ``"seen"`` instead of ``false``.
```